heartbeat/Filesystem

```diff
@@ -355,8 +355,8 @@ list_mounts() {
 		else
 			# <device> on <mountpoint> type <fstype> ...
 			# Use tabs as field separators
-			match_string='\(.*\) on \(.*\) type \([^[:space:]]\)'
-			replace_string="\\1${TAB}\\3${TAB}\\5"
+			match_string='\(.*\) on \(.*\) type \(.*\) .*'
```

Comment:
`[^[:space:]]` should still work. `.*` will match anything.

Reply:
Do you mean:

match_string='\(.*\) on \(.*\) type \([^[:space:]]\+\) .*'

if the match_string doesn't include the mount options and separate it from the capture group, it will be included in the output

Reply:
That sounds good to me.